source/Tutorials/Intermediate/Launch/Creating-Launch-Files.rst

```diff
@@ -292,27 +292,27 @@ To run the launch file created above, enter into the directory you created earli
 
 .. note::
 
-  It is possible to launch a launch file directly (as we do above), or provided by a package.
-  When it is provided by a package, the syntax is:
+  Es posible lanzar un archivo de lanzamiento directamente (como lo hacemos arriba), o proporcionado por un paquete.
```

Comment:
Como dijimos antes, esto me suena mejor:

"It is possible to launch a launch file" -> " Es posible ejecutar un fichero launch"

Reply:
Muchas gracias por la indicación. Estaba inseguro con respecto a como traducir la palabra launch. Lo tomaré en cuenta para futuras páginas.

Tengo una duda con la tradución sugerida aquí, se tiene por covención traducir `file` a `fichero`?. Nosotros en México usamos archivo para referirnos a `file`.

Reply:
Preferiría fichero 🥺